### PR TITLE
Import-Config-from-external-file

### PR DESCRIPTION
**Pull request by oscard0m**
## Description
To move to an external file the configuration of Counting Prawns.

## Expected behavior
* The plugin works as usual.
* To have all the configuration migrated to an external file. 
* Easier to change configuration and made independent from the repository changes.

## Implementation
* The plugin expects to import a configuration file from user's local filesystem called config.json containing the JSON object with all the required parameters to setup the plugin.
* Added configuration file to .gitignore
* Updated README.
## Fixes
* #19

### CODE REVIEW

**Comment by stefanosaittamrf** [comment]
Code is fine. Rebase it.

File: counting_prawns.10s.js [hunk 1]
```diff
@@ -12,16 +12,7 @@
...
-// Configurable params
-const githubConfig = {
-  accessToken: '',
-  username: 'YOUR_USERNAME',
-  owner: 'Marfeel',
-  repos: ['MarfeelXP', 'AliceTenants', 'Gutenberg', 'ProTenants'],
-  color: 'green',
-  alertColor: 'red',
-};
+const githubConfig = require('./config.json');
```
Comment (stefanosaittamrf) [inline counting_prawns.10s.js:15]: You should parse the `json` in order to use it is as an object right?

**Review by stefanosaittamrf — CHANGES_REQUESTED** [review]
Missing the parse

File: counting_prawns.10s.js (hunk 1 above)
Comment (oscard0m) [reply counting_prawns.10s.js:15]: Require of NodeJS resolved the import of a JSON file parsing its content:
https://ide.c9.io/dominguezcelada/require-nodejs-jsonfile

**Comment by oscard0m** [comment]
By the way, once we land this change on master, you can open-spurce this. With a configuration file, this has nothing to do with Marfeel.

**Comment by oscard0m** [comment]
Closed until this repo gets alive again

**Pull request closed without merging**